server/models/Transaction.js

```diff
@@ -12,9 +12,8 @@ export default (Sequelize, DataTypes) => {
 
   const Transaction = Sequelize.define('Transaction', {
     type: DataTypes.STRING, // Expense or Donation
-    description: DataTypes.STRING, // delete #postmigration
+    description: DataTypes.STRING,
     amount: DataTypes.FLOAT, // TODO: change to INTEGER and rename to donationAmount
```

Comment:
why `donationAmount`? Seems weird to have a column in the Transactions table that refers to its name another table (unless it's a foreign key)